doc/maximumflow.md

```diff
@@ -1,9 +1,10 @@
 <!-- AUTOGENERATED. See 'doc/build.jl' for source. -->
-*LightGraphs.jl* provids three algorithms for [maximum flow](https://en.wikipedia.org/wiki/Maximum_flow_problem)
+*LightGraphs.jl* provides four algorithms for [maximum flow](https://en.wikipedia.org/wiki/Maximum_flow_problem)
 computation:
 
```

Comment:
These files are autogenerated. You'll want to change build.jl to make sure this persists. Also, you'll want to ensure that the exported functions are in the list there. Let me know if you have questions.